source/Calamari.Scripting/Calamari.Scripting.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <Nullable>enable</Nullable>
+        <OutputType>Exe</OutputType>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <TargetFrameworks>net452;netcoreapp3.1</TargetFrameworks>
```

Comment:
No net40?

Reply:
I think it has to be net452 for now, Server doesn't support multiple netfx versions atm.... @slewis74 right?

Reply:
If this is a flavour then it should be net40. Server can only support one or other of net40 and net452 for a flavour. For a library we'd add both.